test/runner_test.ts

```diff
@@ -114,22 +121,55 @@ describe('Runner', () => {
           },
           {
             type: 'click',
-            selectors: ['a[href="main2.html"]'],
+            button: 'primary',
+            selectors: ['#test'],
+            offsetX: 1,
+            offsetY: 1,
+          },
+          {
+            type: 'click',
+            button: 'auxiliary',
+            selectors: ['#test'],
+            offsetX: 1,
+            offsetY: 1,
+          },
+          {
+            type: 'click',
+            button: 'secondary',
+            selectors: ['#test'],
+            offsetX: 1,
+            offsetY: 1,
+          },
+          {
+            type: 'click',
+            button: 'back',
+            selectors: ['#test'],
+            offsetX: 1,
+            offsetY: 1,
+          },
+          {
+            type: 'click',
+            button: 'forward',
+            selectors: ['#test'],
             offsetX: 1,
             offsetY: 1,
-            assertedEvents: [
```

Comment:
any reason to remove the navigation assertion? Is it covered by other tests?

Reply:
The navigation assertion isn't necessary since we preventDefault the mouseup (which actually does the navigation). In any case, the goal of this test is to assure the clicks work which this test precisely does by checking mouseup events.

Reply:
Let's still make sure we keep the coverage for the asserted navigation (in case if it's not covered by other existing tests).

Reply:
It's covered in the interactions with OOPIFs test.